src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -135,10 +135,46 @@ def delete_volume(mgr, volname, metadata_pool, data_pools):
     result_str = "metadata pool: {0} data pool: {1} removed".format(metadata_pool, str(data_pools))
     return r, result_str, ""
 
+def _can_renaming_be_done(mgr, volname, newvolname):
```

Comment:
Why aren't these checks a part of the monitor command that handles file system rename?